server/memphis_helper.go

```diff
@@ -503,58 +503,65 @@ func (s *Server) GetMessages(station models.Station, messagesToFetch int) ([]mod
 		messagesToFetch = int(totalMessages)
 	}
 
-	msgs, err := s.memphisGetMsgs(stationName.Intern()+".final",
-		stationName.Intern(),
+	msgs, err := s.memphisGetMsgs(stationName.Intern(),
```

Comment:
Why have you deleted it?

Reply:
Apparently, there was no use of that parameter inside the function, just cleaned it.